source/Halibut/Util/RetryPolicy.cs

```diff
@@ -0,0 +1,63 @@
+using System;
+using System.Diagnostics;
+
+namespace Halibut.Util
+{
+    class RetryPolicy
```

Comment:
nitpick: this could be called "ExponentialBackoffWithJitterRetryPolicy" to give folks a hint what's going on here.

Reply:
It could, but as far as the consumer is concerned it shouldn't know what the retry policy is. If we had an interface as the abstraction then I'd be more inclined to make the concrete class's name a bit more implementation-specific, but at the moment the consuming code just news up a `RetryPolicy` so it shouldn't really care what the implementation is.

If only there were a way we could get something else to provide dependencies to classes...